### PR TITLE
adds prettier

### PR DESCRIPTION
**Pull request by zonzujiro**
TODO:

- [x] add path, which will format all files inside `src`
- [ ] figure out how to format `html` and do not mess up JS objects inside `{% %}`
- [ ] format files with styles. Probably we should migrate to `scss` - https://github.com/prettier/prettier/issues/4948

### CODE REVIEW

File: src/templates/index.html [hunk 1]
```diff
@@ -25,27 +25,15 @@
 og_sitename: Kottans
 fb_admins: 100003342204377, 1744734572, 100000443393783, 100001674114934
 ---
-{% extends "layouts/_layout.html" %}
 
-{% block content %}
-
-{% include "partials/_header.html" %}
+{% extends "layouts/_layout.html" %} {% block content %} {% include
+"partials/_header.html" %}
```
Comment (A-Ostrovnyy) [inline src/templates/index.html]: As for me, it must be more readable.

Comment (zonzujiro) [reply src/templates/index.html]: Pls, see second checkbox in the description of PR.

**Comment by yevhenorlov** [comment]
I think first item on the todo is done - I've updated globs to be more flexible and moved some stuff to separate ignore configs.
@zonzujiro migrating to .scss sounds like a good idea, maybe let's do it in a separate PR? How about moving the todo to the issues?

**Comment by zonzujiro** [comment]
@yevhenorlov fill free to create issues :)

**Comment by yevhenorlov** [comment]
done :) https://github.com/kottans/kottans.github.io/issues/159

**Pull request merged**